### PR TITLE
refactor: cleanup CB related namespaces

### PR DESCRIPTION
**Pull request by jackgerrits**

### CODE REVIEW

File: utl/flatbuffer/vw_to_flat.cc [hunk 1]
```diff
@@ -452,7 +452,7 @@ void to_flat::convert_txt_to_flat(VW::workspace& all)
     {
       if (!VW::example_is_newline(*ae) ||
           (all.example_parser->lbl_parser.label_type == VW::label_type_t::CB &&
-              !CB_ALGS::example_is_newline_not_header(*ae)) ||
+              !VW::example_is_newline_not_header_cb(*ae)) ||
```
Comment (lalo) [inline utl/flatbuffer/vw_to_flat.cc:455]: might as well rename? do we need the _cb?

File: vowpalwabbit/core/include/vw/core/gen_cs_example.h [hunk 2]
```diff
@@ -93,7 +96,7 @@ void gen_cs_example_dm(cb_to_cs& c, VW::example& ec, const VW::cb_label& ld, VW:
       VW::cs_class wc = {0., cl.action, 0., 0.};
 
       // get cost prediction for this action
-      wc.x = CB_ALGS::get_cost_pred<is_learn>(c.scorer, c.known_cost, ec, cl.action, 0);
+      wc.x = VW::get_cost_pred<is_learn>(c.scorer, c.known_cost, ec, cl.action, 0);
```
Comment (lalo) [inline vowpalwabbit/core/include/vw/core/gen_cs_example.h:99]: is this worthy of being in VW:: maybe a cb related seemed more fitting

File: utl/flatbuffer/vw_to_flat.cc (hunk 1 above)
Comment (jackgerrits) [reply utl/flatbuffer/vw_to_flat.cc:455]: This one specifically checks the cb label so its not a generally applicable function. This used to be conveyed by the fact it was in the CB_ALGS namespace but without that i figured a suffix keeps that info around

**Review by lalo — APPROVED** [review]

File: vowpalwabbit/core/include/vw/core/gen_cs_example.h (hunk 2 above)
Comment (jackgerrits) [reply vowpalwabbit/core/include/vw/core/gen_cs_example.h:99]: Discussed offline, punting on taking on more fine grained namespaces

**Pull request merged**